tsconfig.json

```diff
@@ -1,6 +1,9 @@
 {
   "extends": "@chatie/tsconfig",
   "compilerOptions": {
+    "esModuleInterop": true,
```

Comment:
I believe those three options you added have already been set in `@chatie/tsconfig`.

Could you please confirm it and remove those additional settings if possible?